docs/CONTRIBUTING.md

```diff
@@ -512,6 +512,9 @@ one go:
 docker-compose up -d
 ```
 
+> **Note:** set `DEV_DOCKER=1` in a `.env` or `docker-compose.override.yml` 
+file when working with Docker in development.
+
```

Comment:
By updating `docker-compose.ym` directly, the 3 changes in this file should be dropped as well.